feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/presentation/handling/SendingNotificationHandler.kt

```diff
@@ -0,0 +1,35 @@
+package io.novafoundation.nova.feature_push_notifications.data.presentation.handling
+
+import android.content.Context
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.google.firebase.messaging.RemoteMessage
+import io.novafoundation.nova.common.resources.ResourceManager
+import io.novafoundation.nova.feature_push_notifications.R
+
+class SendingNotificationHandler(
+    private val context: Context,
+    notificationManager: NotificationManagerCompat,
+    resourceManager: ResourceManager,
+) : BaseNotificationHandler(
+    notificationManager,
+    resourceManager,
+    R.string.default_notification_channel_id,
+    R.string.default_notification_channel_name,
```

Comment:
Do we need to pass them each time? If we want it to be customizable and avoid passing default value many times we can use default parameters in `BaseNotificationHandler`

Reply:
Yes, I would like to use multiple notification channels depending on the push type